docs/sources/panels-visualizations/visualizations/canvas/index.md

```diff
@@ -107,6 +107,12 @@ When right clicking an element, you are able to edit, delete, duplicate, and mod
 
 {{< figure src="/static/img/docs/canvas-panel/canvas-context-menu-9-2-0.png" max-width="750px" caption="Canvas element context menu" >}}
 
+### Element snapping and alignment
+
+When you're moving elements around the canvas, snapping and alignment guides help you create more precise layouts.
+
```

Comment:
should we add a note here that this feature only works when the panel is not zoomed in at all? (i.e. default scale) cc @drew08t